msteams-platform/messaging-extensions/how-to/link-unfurling.md

```diff
@@ -64,16 +67,9 @@ For a complete manifest example, see [manifest reference](~/resources/schema/man
 
 ## Handle the `composeExtension/queryLink` invoke
 
-After adding the domain to the app manifest, you must update your web service code to handle the invoke request. Use the received URL to search your service and create a card response. If you respond with more than one card, only the first card response is used.
-
-The following card types are supported:
-
-* [Thumbnail card](~/task-modules-and-cards/cards/cards-reference.md#thumbnail-card)
-* [Hero card](~/task-modules-and-cards/cards/cards-reference.md#hero-card)
-* [Office 365 Connector card](~/task-modules-and-cards/cards/cards-reference.md#office-365-connector-card)
-* [Adaptive Card](~/task-modules-and-cards/cards/cards-reference.md#adaptive-card)
```

Comment:
Please revert. Why has this section been deleted. Only line 76 had to be removed.